app/scripts/modules/core/pipeline/config/stages/bake/aws/bakeStage.html

```diff
@@ -71,6 +71,41 @@ <h4 ng-if="viewState.loading" class="text-center">
         <input type="text" class="form-control input-sm"
                ng-model="stage.templateFileName"/>
       </stage-config-field>
+      <stage-config-field label="Extended Attributes" help-key="pipeline.config.bake.extendedAttributes"
+                          ng-if="viewState.roscoMode || (stage.extendedAttributes && _.size(stage.extendedAttributes) > 0)">
```

Comment:
There's a lot of logic in this `ng-if` tag. It would be good to move it into a method on the controller, e.g.
```javascript
this.showExtendedAttributes = () => $scope.viewState.roscoMode || _.size($scope.stage.extendedAttributes);
```
This would keep lodash from leaking into a scope variable and make it more testable (not that we're doing much testing of specific stages right now, but at some point we will).